torch/utils/cpp_extension.py

```diff
@@ -585,8 +627,8 @@ def build_extensions(self) -> None:
             if 'nvcc_dlink' in extension.extra_compile_args:
                 assert self.use_ninja, f"With dlink=True, ninja is required to build cuda extension {extension.name}."
 
-        # Register .cu, .cuh, .hip, and .mm as valid source extensions.
-        self.compiler.src_extensions += ['.cu', '.cuh', '.hip']
+        # Register .cu, .cuh, .hip, .mm and .sycl as valid source extensions.
+        self.compiler.src_extensions += ['.cu', '.cuh', '.hip', '.sycl']
```

Comment:
should we highlight here, the .sycl is not a SYCL std file ext, actually...

Reply:
I left a comment in the last version. Done.